CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   cached model outputs or datasets.
 - Added the following new datasets:
     - `fone`, a Faroese NER dataset, which replaces the previous `wikiann-fo` dataset.
-    - `dansk`, a Danish NER dataset, which complements the previous `dane` dataset.
+    - `dansk`, a Danish NER dataset, which replaces the previous `dane` dataset.
```

Comment:
Oh do we replace it dane? Hmm then that might be worth merging the two (dansk + dane_plus have the same labels). That however seems like a dataset thing. I have added and issue [here](https://huggingface.co/datasets/chcaa/DANSK/discussions/2)

Reply:
Sure thing. I'm quite sure that it won't change the evaluations significantly anyway, given the high correlation, but if you merge them then feel free to open a PR with the new merged dataset 🙂

Reply:
No I think I agree. It is def. not the highest on my todo list for that reason.